### PR TITLE
add GCLOUD_PROJECT to circle config for system tests

### LINKED ISSUE #190: nightly testing flakyness
We haven't had a clean nightly run for spanner for a long time. The fact that we kind of ignore test failures is a sign that something is going wrong. If tests are bad they should be fixed or at least marked to skip, otherwise there is no need to run them at all.

I suggest that we spend some time the following week on our engineering debt and make sure we have several green nightly runs.

Red page that makes me sad: https://circleci.com/gh/googleapis/workflows/nodejs-spanner/tree/master

cc: @crwilcox

### PR DESCRIPTION
**Pull request by kinwa91**
Should fix #190

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -159,6 +159,7 @@ jobs:
           command: npm run system-test
           environment:
             GOOGLE_APPLICATION_CREDENTIALS: .circleci/key.json
+            GCLOUD_PROJECT: long-door-651
```
Comment (JustinBeckwith) [inline .circleci/config.yml]: This is just going to get reverted when synthtool runs next, isn't it?  Can't we just do this for everyone in the templates?  Also, is this covered in kokoro?

**Comment by kinwa91** [comment]
`GCLOUD_PROJECT` is configured for system-tests in [synthtool templates](https://github.com/GoogleCloudPlatform/synthtool/blob/master/synthtool/gcp/templates/node_library/.circleci/config.yml#L163).

nodejs-spanner's synth.py is not configured to pull templates yet, only generate library - I'm going to modify synth.py in a later PR :)

**Comment by kinwa91** [comment]
Merging this first to see if system-test passes

**Comment by kinwa91** [comment]
Merging this first to see if system-test passes

**Comment by kinwa91** [comment]
also kokoro is not properly configured here yet

**Comment by kinwa91** [comment]
Closing in favour of #330

**Pull request closed without merging**